### ISSUE TITLE
Inspection "disconnected from its parent foreach" should check parentS foreach

### ISSUE THREAD

**Issue by machour** [kalessil/phpinspectionsea#1111]
| Subject        | Details                                                                       |
| :------------- | :---------------------------------------------------------------------------- |
| Issue type     | Bug/False-Positive                          |
| Plugin         | Php Inspections (EA Extended) v 3.0.8.1 |
| Language level | PHP 5.6                       |

Current behaviour
--

Inspection wrongly reports that the `$attribute` var is disconnected from the foreach

```php
<?php $newAttrs = $newSearch->getAttrsDef(); ?>
<?php foreach ($newAttrs as $attribute) : ?>
    <ul id="<?= Html::rewriteUrl($attribute->label) ?>">
        <?php foreach ($attribute->values as $value) : ?>
            <li>
                <input type="checkbox" id="cb_<?= $attribute->id ?>_<?= $value->id ?>"
                       name="<?= $attribute->slug ?>[]" value="<?= $value->id ?>"/>
                <label for="cb_<?= $attribute->id ?>_<?= $value->id ?>" <?php if ($value->image_valeur) : ?> style="background-image:url(<?= $value->imageSrc ?>);"<?php endif ?>><?= $value->label ?></label>
            </li>
        <?php endforeach ?>
    </ul>
<?php endforeach; ?>
```

<img width="538" alt="screen shot 2018-11-22 at 10 21 43 am" src="https://user-images.githubusercontent.com/304450/48893351-700cba00-ee40-11e8-95d9-f26b8e2ea04e.png">

Expected behaviour
--

The inspection should not be triggered (maybe it needs to inspect more than one level up?)

Environment details
---

PhpStorm 2018.2.5, build #PS-182.4892.16, buit on October 11, 2018
JRE: 1.9.0, JVM OpenJDK 64bit

Notes
---
- Bugs fixed with the highest priority, so features might be postponed. But it shouldn't stop you, right?
- The project support is powered by community funding via [Patreon](https://www.patreon.com/kalessil)

**Comment by kalessil** [comment]
Good catch @machour, I'll take care of fixing the issue.

**Comment by kalessil** [comment]
Fixed!
